lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1002,12 +1016,28 @@ func validateBumpFeeRequest(in *BumpFeeRequest) (
 func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 	op wire.OutPoint, currentHeight int32) (sweep.Params, bool, error) {
 
-	// Return an error if both deprecated and new fields are used.
+	// Return an error if the bump fee request is invalid.
 	feerate, immediate, err := validateBumpFeeRequest(in)
 	if err != nil {
 		return sweep.Params{}, false, err
 	}
 
+	// In case the user specified a target conf, we estimate the fee rate
+	// for the given target on the fly.
+	if in.TargetConf != 0 {
+		startingFeeRate, err := w.cfg.FeeEstimator.EstimateFeePerKW(
```

Comment:
nit: can move this to the `validateBumpFeeRequest` instead so the returned `feerate` can be used directly here, but no strong opinion.

Reply:
moved